src/transforms/repair.jl

```diff
@@ -0,0 +1,61 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+"""
+    Repair{K}
+
+Perform repairing operation with code `K`.
+
+## Available operations
+
+- K = 0: duplicated vertices and faces are removed
+- K = 1: unused vertices are removed
+- K = 2: non-manifold faces are removed
+- K = 3: degenerate faces are removed
+- K = 4: non-manifold vertices are removed
+- K = 5: non-manifold vertices are split by threshold
+- K = 6: close vertices are merged (given a radius)
+- K = 7: faces are coherently oriented
+"""
+struct Repair{K} <: StatelessGeometricTransform end
+
+# implement operation K = 1
+function apply(transform::Repair{1}, mesh)
+  # get the faces
+  topo = convert(HalfEdgeTopology, topology(mesh))
+  ∂₂₀ = Boundary{2,0}(topo)
+  nelem = nelements(topo)
+
+  # indices of used vertices will be stored in this vector
+  used = Vector{Int}(undef, 0)
```

Comment:
replace this by a Set{Int} instead, which as fast checks for `in`

Reply:
We cannot do `vertices(mesh)[used]` if `used` is a set.